components/icons/DocsArrow.js

```diff
@@ -0,0 +1,7 @@
+export default function DocsArrow({ isDropDown, activeDropDown, onClick, className }) {
+    return (
+        <div className={`w-6 my-auto p-2 rounded-md cursor-pointer ${isDropDown && 'hover:bg-gray-100'}`} onClick={isDropDown ? onClick : () => { }}>
+          {isDropDown && <img src='/img/illustrations/icons/arrow.svg' className={`transition-transform w-fit m-auto duration-200 transform ${className} ${activeDropDown ? 'rotate-90 translate-x-0.5' : ''}`} />}
```

Comment:
Can you please explain what `${className}` is helping us with as I tried without having it and it was working correctly.

Reply:
Yes, in that case it is fine